parity/test_indexer_v_algod.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+import json
+from typing import List
+import yaml
+
+from .json_diff import deep_diff, prettify_diff
+
+NEW, OVERLAP, DROPPED, FULL = "new", "overlap", "dropped", "full"
+DIFF_TYPES = [NEW, OVERLAP, DROPPED, FULL]
+
+# These are the diff reports that will be run and compared/asserted against:
+ASSERTIONS = [DROPPED, FULL]
+
+# Only compare swagger "definitions":
+MODELS_ONLY = True
```

Comment:
Currently, the test only looks at the `definitions:` part of each Swagger. 

* Do we also want to test `responses:` too?
* Anything else we want to check?